### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by VeraXIE1997**
/assign@FeynmanZhou
/assign@JiannaLiu

### CODE REVIEW

File: content/en/observability/_index.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+  content: KubeSphere provides rich observability from infrastructure to applications. It integrates your favorite tools for multi-dimensional monitoring metrics, multi-tenant log query and collection, alerting and notification. Try Kubernetes Observability in KubeSphere to realize visualization in a simple way.
   image: /images/observability/banner.jpg
   showDownload: true
 
 
 image: /images/observability/observability.jpg
 
 section2:
-  title: Discoverability, Observability, Security, Everything You Need in One Platform
+  title: Kubernetes Observability, Discoverability, Security, Everything You Need in One Platform
```
Comment (JiannaLiu) [inline content/en/observability/_index.md:17]: Kubernetes Observability and Monitoring, Everything You Need in One Platform

File: content/en/observability/_index.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
   list:
-    - title: Multi-dimensional Monitoring
+    - title: Multidimensional Kubernetes Monitoring
```
Comment (JiannaLiu) [inline content/en/observability/_index.md:19]: Multi-dimensional

**Review by VeraXIE1997 — COMMENTED** [review]
/assign @JiannaLiu

**Pull request merged**